src/State/Location.cpp

```diff
@@ -254,14 +254,14 @@ void Location::setLocation(const std::string& name)
             unsigned int tileNum = mapFile->elevations()->at(_currentElevation)->floorTiles()->at(i);
             if (tileNum > 1)
             {
-                auto tile = new UI::Tile(tileNum, x, y);
+                auto tile = new UI::Tile(tileNum, Point(x, y));
                 _floor->tiles()->push_back(tile);
             }
 
             tileNum = mapFile->elevations()->at(_currentElevation)->roofTiles()->at(i);
             if (tileNum > 1)
             {
-                auto tile = new UI::Tile(tileNum, x, y - 104);
+                auto tile = new UI::Tile(tileNum, Point(x, y - 104));
```

Comment:
Is there a big performance issue here? What is your point?
If we use Point classes, we should use them in all UI-related code to take advantage of Point arithmetics.